charts/ingress-nginx/values.yaml

```diff
@@ -226,10 +226,10 @@ controller:
   resources:
   #  limits:
   #    cpu: 100m
-  #    memory: 90Mi
+  #    memory: 200Mi
```

Comment:
why are we changing these values?

Reply:
@ChiefAlexander low memory value triggered hpa even w/o any load, so if you install chart with 90Mi memory it will scale hpa to maxreplicas

Reply:
I see. As we don't default the HPA to enabled I'd rather we leave this alone and instead make a note of that in our Charts Readme. What do you think?

Reply:
I don't mind. I will back this value